src/System.Private.CoreLib/src/System/MathF.CoreCLR.cs

```diff
@@ -55,6 +55,7 @@ public static partial class MathF
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern float Floor(float x);
 
+        [Intrinsic]
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern float FusedMultiplyAdd(float x, float y, float z);
```

Comment:
Can these methods have managed implementation?

Reply:
I'm not positive, but I don't believe `InternalCall` works for non-extern methods.